src/main/webapp/app/overview/course-exercises/course-exercises.component.ts

```diff
@@ -15,15 +16,18 @@ import { Exercise, ExerciseService } from 'app/entities/exercise';
 export class CourseExercisesComponent implements OnInit, OnDestroy {
     public readonly DUE_DATE_ASC = 1;
     public readonly DUE_DATE_DESC = -1;
+    public readonly OVERDUE = 0;
```

Comment:
I think it would be better to understand as an enum.
```enum FilterType {
OVERDUE = "OVERDUE", NEEDS_WORK = "NEEDS_WORK"
}```

You can assign enums as a property to components if you want to use them in the template.

If you don't assign strings, the enum members would have numeric values.

Reply:
Refactored order and filter to enums